### PR TITLE
Feature/meta-pool-swaps

### PR DESCRIPTION
**Pull request by rej156**
Prepare for meta pool swap integration

### CODE REVIEW

**Comment by rej156** [comment]
Added `yarn commit` which can be run after git add for git commit messages by wizard for cleaner changelog

**Comment by rej156** [comment]
works now 🥇

File: src/state/swap/hooks.ts [hunk 1]
```diff
@@ -186,6 +186,8 @@ export function useDerivedSwapInfo(
     [currencies]
   )
 
+  // console.log({ isStableSwap })
```
Comment (specialist0x) [inline src/state/swap/hooks.ts:189]: rm

File: src/pages/Swap/index.tsx [hunk 2]
```diff
@@ -150,6 +150,9 @@ export default function Swap() {
       if (isStableSwap) {
         const swapOutputRaw = parsedAmounts?.OUTPUT?.raw
         if (swapOutputRaw) {
+          // console.log({ stableswapTrade })
```
Comment (specialist0x) [inline src/pages/Swap/index.tsx:153]: rm

File: src/state/swap/hooks.ts (hunk 1 above)
Comment (rej156) [reply src/state/swap/hooks.ts:189]:
```suggestion
```

File: src/pages/Swap/index.tsx (hunk 2 above)
Comment (rej156) [reply src/pages/Swap/index.tsx:153]:
```suggestion
```

File: src/hooks/useContract.ts [hunk 3]
```diff
@@ -161,6 +161,7 @@ export function useStableSwapContract(poolName?: StableSwapPoolName, withSignerI
     }
 
     if (isMetaPool(pool.name)) {
+      // console.log({ metaPool })
```
[the remark this answers is not in the document]
Comment (rej156) [reply src/hooks/useContract.ts:164]:
```suggestion
```

**Pull request merged**